sphericart-jax/src/sphericart_jax_cuda.cu

```diff
@@ -37,7 +37,7 @@ std::unique_ptr<C<T>>& _get_or_create_sph_cuda(
 template <template <typename> class C, typename T>
 inline void cuda_sph(cudaStream_t stream, void** in, const char* opaque, std::size_t opaque_len) {
     // Parse the inputs
-    const T* xyz = reinterpret_cast<const T*>(in[0]);
+    T* xyz = reinterpret_cast<T*>(in[0]);
```

Comment:
Can this const be put back in now? I don't think it's a big deal either way

Reply:
fixed